### DISCUSSION TITLE
Navigate to line in file and search

### DISCUSSION THREAD

**Discussion by cpholguera** [community/community#10097] [Mobile] [answered]
Right now when clicking on "View File" from a markdown file a view opens at the top of the file (!), the file is rendered to HTML and there's no option to search. I propose 3 things to improve this experience and see more of the *context* where that changed line lives.

1. It'd be great to be able to "View File" from the current line in the diff.

2. Maybe also include a search option for the opened file? Now we have to scroll until we find the line we are interested in.

3. Markdown files are rendered after clicking "View File". It'd be more useful to view the file as-is. And have an option to switch to the rendered version. In both cases having the possibility to search the content.

![Screenshot_20220113-211037](https://user-images.githubusercontent.com/29175115/149402589-e0ae7e48-e1c7-481b-992b-244c91cc1551.jpg)

**Comment by brianlovin** [comment, marked as answer]
Agreed with these points, thanks for the detailed request @cpholguera — I'll add these notes to our tracking issues to improve file browsing overall, we have a lot of work to do here to make this great!

**Reply by cpholguera** [reply]
You're doing an amazing job! Thanks a lot for adding this as well :)
